manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: WinsiderSS.SystemInformer
+PackageVersion: 3.0.11299.6772

```

Comment:
Package writes `3.0` to the registry, and the GitHub tag is `3.0.6772`. This should either be -
1) Updated to 3.0 -or-
2) Changed to 3.0.6772 and the `AppsAndFeaturesEntries` added with `DisplayVersion`